app/packages/looker-3d/src/state.ts

```diff
@@ -57,6 +57,44 @@ export const isGridOnAtom = atom<boolean>({
   ],
 });
 
+export const gridCellSizeAtom = atom<number>({
```

Comment:
I wish recoil had a way to logically group state... lots of repetition here but just a side comment.